.github/workflows/ci-django-api.yml

```diff
@@ -0,0 +1,237 @@
+        with:
+          linters: pytest
+          run: pytest -ra -vvv ${{ inputs.pytest-parameters }} --cov=. --cov-report=xml
+
+      # Without this workaround Sonar reports a warning about an incorrect source path
+      # sonarcloud-projeect.properties: sonar.python.coverage.reportPaths=coverage.xml
+      - name: Override coverage report source path for Sonar
+        run: sed -i 's@'$GITHUB_WORKSPACE'@/github/workspace/@g' coverage.xml
+
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+  migartions:
+    name: Testing - migrations
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: ${{ inputs.database-image }}:${{ inputs.database-image-version }}
+        env:
+          # Note: not redundant, parent env is not applied to service
+          POSTGRES_USER: db_user
+          POSTGRES_PASSWORD: db_password
+          POSTGRES_DB: test
+          FIELD_ENCRYPTION_KEYS: abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Required by Sonar
+          fetch-depth: 0
+      - name: Setup apt
+        run: |
+          sudo apt-get update
+          sudo apt-get install libpq-dev
+          sudo apt-get install gettext postgresql-client
+          sudo apt-get install gdal-bin
+      - name: Setup Python ${{ inputs.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python-version }}
+          cache: pip
+      - name: Cache pip packages
+        uses: actions/cache@v4
+        env:
+          cache-name: cache-pip-modules
+        with:
+          path: ~/.pip-cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/requirements-dev.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt -r requirements-dev.txt
+      - name: Configure sysctl limits
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
+          sudo sysctl -w vm.max_map_count=262144
+      - name: Check migrations
+        run: |
+          python manage.py makemigrations --dry-run --check
+      - name: Apply migrations
+        run: |
+          python manage.py migrate
+
+  black:
+    name: Coding style - black
```

Comment:
I wonder if we should name these `Style` instead of `Coding style` and then include commitlint to those as well 🤔

Reply:
done